pkg/services/featuremgmt/registry.go

```diff
@@ -1217,6 +1217,14 @@ var (
 			HideFromDocs:      true,
 			HideFromAdminPage: true,
 		},
+		{
+			Name:              "ssoSettingsSAML",
+			Description:       "Enables the SSO settings API and the SAML configuration UIs in Grafana",
```

Comment:
I think this description should be more clear because this flag is enabling the SAML connector for the SSO Settings API.